e2e_test/source/basic/inlcude_key_as.slt

```diff
@@ -59,5 +75,25 @@ select count(rw_key) from upsert_students_default_key
 ----
 15
 
+query I
+SELECT count(*)
+FROM   additional_columns
+WHERE  key_col IS NOT NULL
+       AND partition_col IS NOT NULL
+       AND offset_col IS NOT NULL
+       AND timestamp_col IS NOT NULL
+       AND header_col IS NOT NULL
+----
+101
```

Comment:
How do we know what count to expect here 🤔 

Where can I find the input data

Reply:
https://github.com/risingwavelabs/risingwave/blob/600300a31babec3f4956f088b4f8788cdd104714/scripts/source/prepare_ci_kafka.sh#L79 

it will generate message like

|key|payload|header|
|---|---|---|
|key1|{"a": 1}|[(header1, v1), (header2, v2)]|

Reply:
May mention this in comment to avoid confusion